spec/board_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "spec_helper"
+require "./lib/board"
```

Comment:
Looks like we are not using `lib/board.rb`, but we are using the Board class in `lib/human_player` instead. This test file along with `lib/board.rb` are no longer relevant and can be removed.